.github/workflows/test-action.yml

```diff
@@ -0,0 +1,29 @@
+name: Print PR Details
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  print-pr-details:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+
+      - name: Install GitHub API library
+        run: |
+          python -m pip install --upgrade pip
+          pip install PyGithub
+
```

Comment:
Missing error handling for the Python script execution. Consider adding continue-on-error or error handling steps.